wgpu-core/src/command/render.rs

```diff
@@ -1251,53 +1289,35 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                     count,
                     indexed,
                 } => {
-                    state.is_ready().unwrap();
-
-                    let name = match (count, indexed) {
-                        (None, false) => "drawIndirect",
-                        (None, true) => "drawIndexedIndirect",
-                        (Some(..), false) => "multiDrawIndirect",
-                        (Some(..), true) => "multiDrawIndexedIndirect",
-                    };
+                    state.is_ready()?;
 
                     let stride = match indexed {
                         false => 16,
                         true => 20,
                     };
 
                     if count.is_some() {
-                        assert!(
-                            device.features.contains(wgt::Features::MULTI_DRAW_INDIRECT),
-                            "The feature MULTI_DRAW_INDIRECT must be enabled to use {}",
-                            name
-                        );
+                        if !device.features.contains(wgt::Features::MULTI_DRAW_INDIRECT) {
+                            return Err(RenderPassError::MissingDeviceFeature(
```

Comment:
let's use the actual bitflags in the error instead of string slices. Users can always convert them to strings if they need that.